packages/test_express/package.json

```diff
@@ -21,7 +21,7 @@
     "cls-hooked": "^4.2.2",
     "eslint": "^4.18.2",
     "express": "^4.16.4",
-    "mocha": "^3.0.2"
+    "mocha": "^3.5.3"
```

Comment:
For this package if mocha upgrade to a newer version than `3.5.3` then the CI test will freeze. We need to find the root cause and upgrade mocha to the same as other packages.